Translation/en/RedirectedResources/assets/abdata/communication/info_00/topictalkrare_02/zz_machineTranslation.txt

```diff
@@ -1,117 +1,138 @@
-「ホエールウオッチングに興味があるんだよね」=“I'm interested in whale-watching.”
```

Comment:
Something bad happened to this file, the lines are completely different in the new version.